integration-tests/migration/upgrade_version_test.go

```diff
@@ -3,9 +3,10 @@ package migration
 import (
 	"testing"
 
-	"github.com/smartcontractkit/chainlink-testing-framework/utils"
 	"github.com/stretchr/testify/require"
 
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+
 	"os"
```

Comment:
std lib import - did it miss this one?

Reply:
That is a bit odd.

Reply:
Touched the file and then it decided it needed fixed. Weird why it didn't to it before.